contracts/AcceleratingDistributor.sol

```diff
@@ -99,10 +77,36 @@ contract AcceleratingDistributor is ReentrancyGuard, Ownable, Multicall {
     );
     event Exit(address indexed token, address indexed user, uint256 tokenCumulativeStaked);
 
+    /**************************************
+     *           STATE VARIABLES          *
+     **************************************/
+
+    IERC20 public immutable rewardToken;
+
+    mapping(address => StakingToken) public stakingTokens;
+
+    /**************************************
+     *              MODIFIERS             *
+     **************************************/
+
+    modifier onlyEnabled(address stakedToken) {
+        require(stakingTokens[stakedToken].enabled, "stakedToken not enabled");
+        _;
+    }
+
+    modifier onlyInitialized(address stakedToken) {
+        require(stakingTokens[stakedToken].lastUpdateTime != 0, "stakedToken not initialized");
+        _;
+    }
+
     /**************************************
      *          ADMIN FUNCTIONS           *
      **************************************/
 
+    constructor(address _rewardToken) {
```

Comment:
I would move `constructor` above `Admin Functions` and below `Modifiers`

Reply:
Done. I added a CONSTRUCTOR banner to separate it from the modifiers section. I think I'll add documentation for the constructor in the "missing docstrings" commit.